### PR TITLE
chore(log): create directories for query log

### LINKED ISSUE #9161: Is `ENABLE_QUERY_LOG_FILE` still useful or an obsolete variable?
`const ENABLE_QUERY_LOG_FILE: bool = false;`
only appears at https://github.com/risingwavelabs/risingwave/blob/main/src/utils/runtime/src/lib.rs#L40-L179 once.

And it seems `query_log_path` can totally overtake the responsibility.

Also, another issue is whether we expect RW to create the directory for `RW_QUERY_LOG_PATH`/`query_log_path`, or whether the user should create it.

We tried to enable it in `kube-bench`, but
```
thread 'main' panicked at 'failed to create '.risingwave/log/query.log': No such file or directory (os error 2)', src/utils/runtime/src/lib.rs:188:17
```

### PR DESCRIPTION
**Pull request by lmatz**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

close #9161

enable https://github.com/risingwavelabs/kube-bench/pull/126

## Checklist For Contributors

~- [ ] I have written necessary rustdoc comments~
~- [ ] I have added necessary unit tests and integration tests~
~- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).~
~- [ ] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)~
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

~- [ ] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.~

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: src/utils/runtime/src/lib.rs [hunk 1]
```diff
@@ -180,6 +180,12 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
     if query_log_path.is_ok() || ENABLE_QUERY_LOG_FILE {
         let query_log_path = query_log_path.unwrap_or(".risingwave/log".to_string());
```
Comment (fuyufjh) [inline src/utils/runtime/src/lib.rs:181]: `.risingwave/log` seems to be designed for risedev. How about just writing the log files to `./` and set `RW_QUERY_LOG_PATH` in risedev?

Comment (fuyufjh) [reply src/utils/runtime/src/lib.rs:181]: Hmm, one additional thing: 

https://github.com/risingwavelabs/risingwave/blob/a707bc215f69468e793f2d6fa34a1eb62ba9fec4/src/utils/runtime/src/lib.rs#L132-L133

This is a little bit dangerous, especially considering we are going to enable slow query logs in Cloud. I think we had better simply use 2 independant vairables:

1. If `RW_QUERY_LOG_PATH` is set, use it as path for log files (both query log and slow query log).
2. If `ENABLE_QUERY_LOG_FILE` is set to `true`, enable query log. (Slow query is always enabled)

Comment (fuyufjh) [reply src/utils/runtime/src/lib.rs:181]: cc. @xxchan

Comment (xxchan) [reply src/utils/runtime/src/lib.rs:181]: LGTM. BTW, the query log feat was added just for ad-hoc debugging without deliberations. 😇

So any enhancements to make it more mature should be good.

Comment (fuyufjh) [reply src/utils/runtime/src/lib.rs:181]: One little enhancement is: why `RW_QUERY_LOG_PATH` has a prefix `RW_` but `ENABLE_QUERY_LOG_FILE` does not 🤣

Comment (lmatz) [reply src/utils/runtime/src/lib.rs:181]: `ENABLE_QUERY_LOG_FILE` 
it is a constant

If `env` variable, then `RW_` I suppose

**Review by fuyufjh — APPROVED** [review]
LGTM

**Comment by lmatz** [comment]
seems enabling query log by default in Risedev causes log file too big error

**Pull request merged**